client/startup/unread.js

```diff
@@ -30,7 +32,8 @@ Meteor.startup(() => {
 		let unreadAlert = false;
 
 		const unreadCount = fetchSubscriptions().reduce((ret, subscription) => {
-			fireGlobalEvent('unread-changed-by-subscription', subscription);
+			const room = ChatRoom.findOne({ _id: subscription.rid }, { fields: { usersCount: 1 } });
```

Comment:
@ggazzo can you please validate if doing this find for _each_ room every time _any_ subscription changes is something heavy on the client?

Reply:
I dont think so,  but only because the collection has a index by id (and only by id)... I dont know, but this `Tracker.autorun` and fire a trigger " for each room every time any subscription changes " sounds weird/wrong..